seed/breakfast-seeder.js

```diff
@@ -1,7 +1,7 @@
 const Breakfast = require('../models/breakfast');
 const mongoose = require('mongoose');
 
-mongoose.connect('mongodb://localhost:27017/chicken-fries',{ useNewUrlParser: true });
+mongoose.connect(process.env.DB_HOST+'://'+ process.env.DB_USER +':'+ process.env.DB_PASS +'@ds163402.mlab.com:63402/chicken-fries',{ useNewUrlParser:true});
```

Comment:
Requirements: Duplicate logic. Consider that multiple seed files are using this single line of code, Is there a way you can write this once and invoke it multiple times by all the seeds?

Reply:
create issue, didn't fix it yet https://github.com/piazzaana/chicken-fries/issues/64